xsimlab/stores.py

```diff
@@ -73,7 +73,10 @@ def default_fill_value_from_dtype(dtype=None):
 
 def get_auto_chunks(shape, dtype):
     # A hack to get chunks guessed by zarr
-    arr = zarr.create(shape, dtype=dtype)
+    if(dtype == object or dtype == np.object):
```

Comment:
First comparison would be enough

```suggestion
    if dtype == object:
```